### ISSUE TITLE
total Error Count  axe-core result 

### ISSUE THREAD

**Issue by asthaawasthi-laitkor** [dequelabs/axe-core#4307]
### Product

axe-core

### Question

from the output json result file , I am including incomplete and violation nodes as an error, but the total count of error is very less, I compared the total error count with other  API , please specify the possible reason .

**Comment by straker** [comment]
Thanks for the question. Could you provide an example a page that provides different total counts from axe-core and the other APIs, and the code you're using to do the counting?

**Comment by asthaawasthi-laitkor** [comment]
attaching the CLI Command 
```
/root/.nvm/versions/node/v20.10.0/bin/axe --chrome-options="no-sandbox" https://www.villaserenaseniorapartments.com/gallery/ --timeout=120 --save all.json --tags wcag2a,wcag2aa,wcag2aaa,wcag21,wcag21a,wcag21aa,wcag22aa,best-practice,wcag2a-obsolete,wcag***,ACT,section508,section508.*.*,TTv5,TT*.*,EN-301-549,EN-9.*,experimental,cat.*,wcag111,wcag412,cat.forms,wcag135,EN-9.1.3.5  --disable region  --rules autocomplete-valid
```
[Uploading all.json…]()

json Result 

from the result json i am including all nodes of violation  in my total error count.

**Comment by asthaawasthi-laitkor** [comment]
JSON Output 
 [all.json](https://github.com/dequelabs/axe-core/files/14034158/all.json)

**Comment by straker** [comment]
When you say you compare the results of the count with other APIs, what do you mean? Do you mean you run the axe DevTools Chrome extension on the page and it returns a different total issues count that the axe CLI? 

If that's the case it's because the extension runs a certain subset of all the rules that the axe CLI (or axe-core for that matter) runs. The extension runs all WCAG 2.1 A and AA, WCAG 2.0 A and AA. Best practices rules, incomplete rules (needs review) and experimental rules are turned off. From the code snippet you provided, that means the CLI is running more rules than the extension is.

When I run the extension on the provided page, it returns 6 total issues. However when I run with your settings from the provided code snippet it will return 100+ issues. If you update your snippet to run the exact same tags as the extension and only look at the `violation` nodes then they will return the same total issues.

```js
const results = await axe.run({ runOnly: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa'] })
const totalIssues = results.violations.reduce((count, violation) => {
  return count + violation.nodes.length
}, 0);

console.log(totalIssues);  // 6
```

**Comment by asthaawasthi-laitkor** [comment]
Thank you for the information .
we are scanning the above attached code on CLI and save the output jsson as final output file.
from the document https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#axe-core-tags, I am Passing all tags with wcag ,'wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa' ,wcag2aaa, wcag22aa, so will it check for AAA level (Version 2.0),

**Comment by asthaawasthi-laitkor** [comment]
Sorry for the confusion ,
let me clear the question ,

**Comment by straker** [comment]
Axe-core and Userway Web Accessibility Checker have different rules that are run and different ways of interpreting results. Axe-core's policy is that of zero false positive, so we will not report an error if we think it could be a false positive. Instead we move those to Needs Review (`incomplete`) in order to inform you that manual testing is required to determine if it's a violation. 

Axe-core has 3 AAA rules, which you can find more information here https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#wcag-2x-level-aaa-rules

**Comment by asthaawasthi-laitkor** [comment]
Thanks @straker  for your valuable insight .
